ipi/inputs/normalmodes.py

```diff
@@ -159,6 +159,24 @@ class InputNormalModes(Input):
                 "options": ["exact", "cayley", "bab"],
             },
         ),
+        "fft_threads": (
```

Comment:
Maybe add some guidance on how to set it appropriately on an HPC, considering setting it appropriately can speed up PIMD a fair bit. Is it possible to provide an auto mode which sets it to the number of threads available (OMP_NUM_THREADS)?

Reply:
I tested a bit and it's not trivial, you can "overparallelize" particularly for small number of beads. I think for the moment it's best to leave it as a manual optimization